wiki/Guides/How_to_Use_the_Offset_Wizard/fr.md

```diff
@@ -1,23 +0,0 @@
-# Comment utiliser l'assistant de décalage audio
```

Comment:
why are you removing these files? they need to be marked with `outdated: true` in a separate PR instead